tests/components/ping/test_device_tracker.py

```diff
@@ -1,15 +1,31 @@
 """Test the binary sensor platform of ping."""
+from datetime import timedelta
+import os
+from unittest.mock import patch
 
 import pytest
 
+from homeassistant.components.device_tracker import legacy
 from homeassistant.components.ping.const import DOMAIN
+from homeassistant.const import EVENT_HOMEASSISTANT_STARTED
 from homeassistant.core import DOMAIN as HOMEASSISTANT_DOMAIN, HomeAssistant
 from homeassistant.helpers import entity_registry as er, issue_registry as ir
 from homeassistant.setup import async_setup_component
 
 from tests.common import MockConfigEntry
 
 
+@pytest.fixture(name="yaml_devices")
+def mock_yaml_devices(hass):
```

Comment:
I can't use `mock_device_tracker_conf ` here as I'm directly reading the yaml via `load_yaml_config_file` instead of using `async_get_config`. Anyway, I changed this to make use of `patch_yaml_files ` instead of relaying on writing files.